src/CliWrap.FSharp/CliWrap.FSharp.fsproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net9.0</TargetFramework>
+		<TargetFramework>netstandard2.0</TargetFramework>

```

I think we'll want to multi-target, this is what [CliWrap does](https://github.com/Tyrrrz/CliWrap/blob/006be6c019668d790cff03402627645ddeb51f83/CliWrap/CliWrap.csproj#L4) and also follows [Microsoft recommendations](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#when-to-target-netx0-vs-netstandard).

> Most widely used libraries multi-target for both .NET Standard 2.0 and .NET 5+. Supporting .NET Standard 2.0 gives you the most reach, while supporting .NET 5+ ensures you can leverage the latest platform features for customers that are already on .NET 5+.

```suggestion
		<TargetFrameworks>netstandard2.0;netstandard2.1;net8.0;net9.0</TargetFrameworks>
```

---

Do we know if there is any difference in the IL or if the JIT/AOT would process this netstandard2.0 assembly any differently?

I think we want netstandard2.1 for the null annotations but I'm unsure that net8/net9 wouldn't be just bloat?

---

I'm not intimately familiar with the differences, but I think AOT is the main reason.